-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-dd4529/33276/summary.html
**COMMIT**: 7d032c6de49ada202c497450a70b0b35a8e5d813
**CMSSW**: CMSSW_13_2_X_2023-06-20-1100/el8_amd64_gcc11
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/42027/33276/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_PathStatusFilter.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_Prescaler.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_main.cc
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_ModuloEventIDFilter.cc:4:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_ModuloEventIDFilter.cc: In function 'void ____C_A_T_C_H____T_E_S_T____4()':
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_ModuloEventIDFilter.cc:65:52: error: no matching function for call to 'edm::test::TestProcessor::TestProcessor(const string&)'
   65 |   REQUIRE_THROWS_AS(edm::test::TestProcessor(config), cms::Exception);
      |                                                    ^
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/Modules/test/test_catch2_ModuloEventIDFilter.cc:1:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_2_X_2023-06-20-1100/src/FWCore/TestProcessor/interface/TestProcessor.h:165:7: note: candidate: 'edm::test::TestProcessor::TestProcessor(const Config&, edm::ServiceToken)'
  165 |       TestProcessor(Config const& iConfig, ServiceToken iToken = ServiceToken());
</pre>

--- a/FWCore/Integration/test/EDAlias_t.cpp
+++ b/FWCore/Integration/test/EDAlias_t.cpp
@@ -175,7 +175,7 @@ process.moduleToTest(process.test, cms.Task(process.intprod))
 }
 
 TEST_CASE("Configuration with all products of a module with a given product instance name", s_tag) {
-  const std::string baseConfig{
+  constexpr const std::string_view baseConfig{
       R"_(from FWCore.TestProcessor.TestProcess import *
 import FWCore.ParameterSet.Config as cms
 
@@ -313,7 +313,7 @@ process.moduleToTest(process.test, cms.Task(process.intprod, process.intprod2))
 
 ////////////////////////////////////////
 TEST_CASE("No products found with wildcards", s_tag) {
-  const std::string baseConfig{
+  constexpr const std::string_view baseConfig{
       R"_(from FWCore.TestProcessor.TestProcess import *
 import FWCore.ParameterSet.Config as cms
 

